lib/node_modules/@stdlib/stats/base/snanvariancepn/src/main.c

```diff
@@ -30,20 +32,19 @@
 *
 * -   Higham, Nicholas J. 1993. "The Accuracy of Floating Point Summation." _SIAM Journal on Scientific Computing_ 14 (4): 783–99. doi:[10.1137/0914050](https://doi.org/10.1137/0914050).
 *
-* @param N       number of indexed elements
-* @param W       two-element output array
-* @param X       input array
-* @param stride  stride length
-* @return        output value
+* @param N        number of indexed elements
+* @param X        input array
+* @param strideX  stride length
+* @param offsetX  starting index
+* @param n        pointer
+* @return         output value
 */
-static void snansumpw( const int64_t N, double *W, const float *X, const int64_t stride ) {
-	int64_t ix;
-	float *xp1;
-	float *xp2;
+static float snansumpw( const CBLAS_INT N, const float *X, const CBLAS_INT strideX, const CBLAS_INT offsetX, CBLAS_INT *n ) {
```

Comment:
@gururaj1512 Rather than make these changes to this internal function, why not just go ahead and add the package `blas/ext/base/snannsumpw` in a separate PR and then use that API here after it is merged? While the changes here are okay, they are just going to get thrown away once we have `snannsumpw`. This is what we already do in `dnanvariancepn` and would be good to follow a similar approach here.